R/calc_plan_stats.R

```diff
@@ -0,0 +1,88 @@
+calc_plan_stats <- function(redist_plans, redist_map, ...) {
+
+    redist_plans <- redist_plans %>%
+        mutate(total_vap = tally_var(redist_map, vap),
+               plan_dev =  plan_parity(redist_map),
+               comp_edge = distr_compactness(redist_map),
+               comp_polsby = distr_compactness(redist_map,
+                                               measure = "PolsbyPopper"),
+               ndv = tally_var(redist_map, ndv),
```

Comment:
Also, to avoid CRAN errors, any variable that lives inside the data frame (here, `ndv`) should be prefixed by `.data$`.  So here `tally_var(redist_map, .data$ndv)`

lines 4, 10, 11 also need this fix